build/azDevops/azure/azure-pipelines-javaspring-k8s.yml

```diff
@@ -62,6 +62,7 @@ variables:
   self_repo_src: java
   self_repo_tf_src: deploy/azure/app/kube
   self_generic_name: stacks-api
+  self_pipeline_repo: stacks-pipeline-templates
```

Comment:
This is relating to another branch, best move it to the other branch before merging.